sphinx_gallery/gen_gallery.py

```diff
@@ -101,6 +101,7 @@ def generate_gallery_rst(app):
 
 
 gallery_conf = {
+    'filename_pattern': os.path.sep + 'plot',
```

Comment:
Same remark about os.path.sep -> os.sep

Also I think we need to do `re.escape(os.sep)` so that backslashes are escaped properly on Windows.